test/test-ghe-backup.sh

```diff
@@ -524,3 +524,26 @@ begin_test "ghe-backup with no leaked keys"
 
 )
 end_test
+
+begin_test "ghe-backup honours --version flag"
+(
+  set -e
+
+  # Make sure a partial version string is returned
+  ghe-backup --version | grep "GitHub backup-utils v"
+
+)
+end_test
+
+begin_test "ghe-backup honours --help and -h flags"
+(
+  set -e
+
+  arg_help=`ghe-backup --help | grep -o 'Usage: ghe-backup'`
+  arg_h=`ghe-backup -h | grep -o 'Usage: ghe-backup'`
```

Comment:
We prefer `$()` over backticks. 

Reminder to myself: add this to the styleguide.

Reply:
Good to know! Fixed.